functions/fetch-mixer-names/fetch-mixer-names.js

```diff
@@ -0,0 +1,36 @@
+const fetch = require('node-fetch');
+
+exports.handler = async (event, context) => {
+  try {
+    const streamerMarkdown = await fetch(
+      'https://raw.githubusercontent.com/ChristopherBiscardi/party-corgi/master/docs/list-of-streamers.md'
+    )
+      .then(res => res.text())
+      .then(body => {
+        return body;
+      });
+
+    const mixerRegex = new RegExp(
+      /(https?:\/\/(.+?\.)?mixer\.com(\/[A-Za-z0-9\-\._~:\/\?#\[\]@!$&'\(\)\*\+,;\=]*)?)/g
```

Comment:
hmm, it would probably be good to have some structured data in the future so we don't have to do this. (this is fine for this PR though)

Reply:
@ChristopherBiscardi Could you elaborate? I'm not sure I follow...

Reply:
The regex is long and unwieldy for people that aren't used to reading regexes. It would be nice in the future if we integrated the discord bot so that it sent structured data (like JSON) into this repo and we could get rid of the regex

Reply:
@ChristopherBiscardi Agreed! We can definitely make an improvement there. I have to Google my Regexes **every** time 🤣